wsi/drm/drmdisplay.cpp

```diff
@@ -154,6 +154,49 @@ void DrmDisplay::DrmConnectorGetDCIP3Support(
   return;
 }
 
+void DrmDisplay::GetEDIDDisplayData(const ScopedDrmObjectPropertyPtr &props) {
+  uint8_t *edid = NULL;
+  uint64_t edid_blob_id;
+  struct edid_display_data display_data[4];
+  drmModePropertyBlobPtr blob;
+
+  GetDrmObjectPropertyValue("EDID", props, &edid_blob_id);
+  blob = drmModeGetPropertyBlob(gpu_fd_, edid_blob_id);
+  if (!blob) {
+    return;
+  }
+
+  edid = (uint8_t *)blob->data;
+  if (!edid) {
+    return;
+  }
+  std::memset(display_data, 0, sizeof(display_data));
+  std::memcpy((void *)display_data, (void *)(edid + 54),
+              sizeof(edid_display_data) * 4);
+
+  for (int i = 0; i < 4; i++) {
+    if (!(display_data[i].indicate == 0x0000 &&
+          display_data[i].reserved1 == 0x00 &&
+          display_data[i].reserved2 == 0x00))
+      continue;
+
+    if (display_data[i].tag_number == 0xfc) {
+      display_name_.clear();
+      display_name_.assign((char *)display_data[i].desc_data, 13);
+    }
+  }
+
+  ITRACE("EDID Display Name: %s\n", display_name_.c_str());
+}
+
+bool DrmDisplay::CheckLimitedMonitor() {
```

Comment:
pls add comment for the aim of this piece of code  is that to w/a certain display which is not capable for the retrieved modes it declares.

Reply:
And we still can have those fixed constant defined by MACRO, it is more meaningful, right

Reply:
> And we still can have those fixed constant defined by MACRO, it is more meaningful, right

agree